manifests/app-proxy/app-proxy.crb.yaml

```diff
@@ -1,13 +1,14 @@
 apiVersion: rbac.authorization.k8s.io/v1
-kind: RoleBinding
+kind: ClusterRoleBinding
 metadata:
   labels:
     app: cap-app-proxy
   name: cap-app-proxy
 roleRef:
   apiGroup: rbac.authorization.k8s.io
-  kind: Role
-  name: cap-app-proxy
+  kind: ClusterRole
+  name: argo-server-cluster-role
 subjects:
   - kind: ServiceAccount
     name: cap-app-proxy
+    namespace: default
```

Comment:
how will this work?

Reply:
I tested it locally - I coped this file to the local clone of my runtime repo, and ran `kustomize build` - the resulting manifests contained this CRD with the correct namespace of my runtime (the one defined in the `kustomization.yaml` file).

Reply:
oh, right, kustomize replaces `default` with the compiled namespace